src/test/java/org/helpberkeley/memberdata/MainTest.java

```diff
@@ -605,22 +681,6 @@ public void oneKitchenDriverHistoryTest() throws IOException, CsvException {
         Main.main(args);
     }
 
-    @Test
-    public void restaurantTemplateTest() throws IOException, CsvException {
-        String[] args = { Options.COMMAND_RESTAURANT_TEMPLATE };
-        Main.main(args);
-        assertThat(WorkRequestHandler.getLastStatusPost()).isNotNull();
-        assertThat(WorkRequestHandler.getLastStatusPost().raw).contains("Status: Succeeded");
-    }
-
-    @Test
-    public void oneKitchenRestaurantTemplateTest() throws IOException, CsvException {
-        String[] args = { Options.COMMAND_ONE_KITCHEN_RESTAURANT_TEMPLATE };
-        Main.main(args);
-        assertThat(WorkRequestHandler.getLastStatusPost()).isNotNull();
-        assertThat(WorkRequestHandler.getLastStatusPost().raw).contains("Status: Succeeded");
-    }
-
```

Comment:
Now that we have a handful of more explicit template tests, these didn't seem to be needed anymore.